### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by LanaDX**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,20 +3,42 @@
...
+The [ASPxPopupControl](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl) does not have the built-in capability to resize nested controls when it is resized. This example illustrates how to automatically resize controls places inside the popup.
```
Comment (arminal) [inline Readme.md]:
```suggestion
The [ASPxPopupControl](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl) does not have a built-in capability to resize nested controls when it is resized. This example illustrates how to automatically resize control places inside a popup. 
```

File: Readme.md [hunk 2]
```diff
@@ -3,20 +3,42 @@
...
+The [ASPxPopupControl](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl) does not have the built-in capability to resize nested controls when it is resized. This example illustrates how to automatically resize controls places inside the popup. 
 
-<br/>
+To accomplish this task, execute the following steps:
 
+1. Set the [ASPxPopupControl.ScrollBars](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControlBase.ScrollBars) property to `Auto`. This option allows the popup control to evaluate the content width and height.
```
Comment (arminal) [inline Readme.md]:
```suggestion
1. Set the [ASPxPopupControl.ScrollBars](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControlBase.ScrollBars) property to `Auto`. This option allows the popup control to evaluate content width and height.
```

File: Readme.md [hunk 3]
```diff
@@ -3,20 +3,42 @@
...
+    ```html
+    <dx:ASPxPopupControl ID="ASPxPopupControl1" runat="server" Height="500px" Width="500px" 
+                         ScrollBars="Auto" AllowResize="true" LoadContentViaCallback="OnFirstShow">
+        ...
+    </dx:ASPxPopupControl>
+    ```
+2. Handle the [ASPxClientPopupControl.AfterResizing](https://docs.devexpress.com/AspNet/js-ASPxClientPopupControlBase.AfterResizing) event. In the event handler, call the [AdjustControl](https://docs.devexpress.com/AspNet/js-ASPxClientControl.AdjustControl) method for a nested control to force it to reevaluate its size.
```
Comment (arminal) [inline Readme.md]: In the event handler, call the [AdjustControl](https://docs.devexpress.com/AspNet/js-ASPxClientControl.AdjustControl) method for a nested control to force it [it what: control or method?] to reevaluate its size.

**Comment by arminal** [comment]
#Articles_Grammar

File: Readme.md (hunk 3 above)
Comment (LanaDX) [reply Readme.md]:
```suggestion
2. Handle the [ASPxClientPopupControl.AfterResizing](https://docs.devexpress.com/AspNet/js-ASPxClientPopupControlBase.AfterResizing) event. In the event handler, call the [AdjustControl](https://docs.devexpress.com/AspNet/js-ASPxClientControl.AdjustControl) method for a nested control to force the control to reevaluate its size.
```

**Pull request merged**